src/wp-includes/user.php

```diff
@@ -4165,6 +4165,101 @@ static function ( $item ) use ( $reserved_names ) {
 	);
 }
 
+/**
+* Registers the personal data eraser for users.
+*
+* @since 6.7.0
```

Comment:
6.7.0 wouldn't be accurate here, but we can't say for sure if it would be 7.0 or some other version. Typically in this case, the recommended tag is something like this (which gets changed when committed to whichever version is being targeted):

I'd update all the other `@since` tags in this PR to match.

```suggestion
* @since n.e.x.t
```